cmd/kor/root.go

```diff
@@ -50,19 +49,26 @@ func Execute() {
 	rootCmd.PersistentFlags().BoolVar(&opts.NoInteractive, "no-interactive", false, "Do not prompt for confirmation when deleting resources. Be careful using this flag!")
 	rootCmd.PersistentFlags().BoolVarP(&opts.Verbose, "verbose", "v", false, "Verbose output (print empty namespaces)")
 	addFilterOptionsFlag(rootCmd, filterOptions)
+}
 
+func Execute() {
+	_ = rootCmd.ParseFlags(os.Args)
 	if err := filterOptions.Validate(); err != nil {
 		fmt.Fprintf(os.Stderr, "Error while validating filter options '%s'", err)
 		os.Exit(1)
 	}
+	filterOptions.Modify()
 	if err := rootCmd.Execute(); err != nil {
 		fmt.Fprintf(os.Stderr, "Error while executing your CLI '%s'", err)
 		os.Exit(1)
 	}
 }
 
-func addFilterOptionsFlag(cmd *cobra.Command, opts *kor.FilterOptions) {
-	cmd.PersistentFlags().StringVarP(&opts.ExcludeLabels, "exclude-labels", "l", opts.ExcludeLabels, "Selector to filter out, Example: --exclude-labels key1=value1,key2=value2.")
+func addFilterOptionsFlag(cmd *cobra.Command, opts *filters.Options) {
+	cmd.PersistentFlags().StringVarP(&opts.ExcludeLabels, "exclude-labels", "l", opts.ExcludeLabels, "Selector to filter out, Example: --exclude-labels key1=value1,key2=value2. If --include-labels is set, --exclude-labels will be ignored.")
```

Comment:
why not allow both to exist together? maybe I have a bunch of objects that generally I want to include but there are some exceptions I want to exclude with some extra set of labels.

Reply:
1. I want to be consistent with the current ns processing. 
2. both hoping and hoping for a less effective approach.
ps: The current way of initiating requests by namespace is also very inefficient, causing unnecessary pressure on the apiserver, although in many cases the number of ns is not large.